skimage/exposure/tests/test_exposure.py

```diff
@@ -370,7 +370,7 @@ def test_adapthist_grayscale():
     adapted = exposure.equalize_adapthist(img, kernel_size=(57, 51),
                                           clip_limit=0.01, nbins=128)
     assert img.shape == adapted.shape
-    assert_almost_equal(peak_snr(img, adapted), 102.019, 3)
+    assert_almost_equal(peak_snr(img, adapted), 100.140, 3)
```

Comment:
This seems like a major drop... Do you know why there is a difference?

Reply:
@jni I made the same comment, I copy here Marvin's answer (which is one of the "resolved" blocks): "As mentioned in the description, the PR fixes border artefacts which are gone here but present in master. This would be reflected in this value no? By contrast, in the test test_adapthist_alpha, where the image size is a multiple of the kernel size and both the nd branch and master don't exhibit artefacts, values stay the same."

Reply:
Thanks a lot @emmanuelle for forwarding the comment :)